libs/designer/src/lib/core/clonetostandard/utils/cloneStrings.ts

```diff
@@ -5,34 +5,39 @@ export const useCloneStrings = () => {
 
   return {
     sourceSectionTitle: intl.formatMessage({
-      defaultMessage: 'Source: Logic Apps Consumption',
-      id: '3+rUhW',
+      defaultMessage: 'Consumption logic app',
+      id: 'iMCTbJ',
       description: 'Title for the source section',
     }),
     sourceDescription: intl.formatMessage({
-      defaultMessage: 'Source specifications are automatically populated based on your consumption resource.',
-      id: 'ql/lcC',
+      defaultMessage: 'Source details are automatically populated based on your Consumption logic app resource.',
+      id: 'ozl/lK',
       description: 'Description for the source section',
     }),
     destinationSectionTitle: intl.formatMessage({
-      defaultMessage: 'Destination: Logic Apps Standard',
-      id: 'Pnedy5',
+      defaultMessage: 'Standard logic app',
+      id: 'uoSee3',
       description: 'Title for the destination section',
     }),
     destinationDescription: intl.formatMessage({
-      defaultMessage: 'Select the destination to clone your consumption resources to.',
-      id: 'GtXpHi',
+      defaultMessage: 'Provide details for the destination Standard logic app.',
+      id: '0L+QZU',
       description: 'Description for the destination section',
     }),
-    newWorkflowName: intl.formatMessage({
-      defaultMessage: 'New workflow name',
-      id: 'MZXs6p',
-      description: 'Label for new workflow Name field',
-    }),
     workflowNameDescription: intl.formatMessage({
       defaultMessage: 'Avoid spaces and the following symbols in your workflow name: \\ / : * ? " < > | @, #, $, %, &',
       id: 'ZbeL1D',
       description: 'Description for workflow name field and the expected format of the name.',
     }),
+    sourceLogicAppDescription: intl.formatMessage({
+      defaultMessage: 'Keep or edit the default name for the destination Standard logic app.',
```

Comment:
Source or destination? The object label and descriptionr reference the source (Consumption) logic app but the default message references the destination Standard logic app?

@Elaina-Lee @bonicaayala